docs/reference/commandline/network_create.md

```diff
@@ -134,7 +135,13 @@ The following are those options and the equivalent docker daemon flags used for
 | `com.docker.network.bridge.enable_icc`           | `--icc`     | Enable or Disable Inter Container Connectivity        |
 | `com.docker.network.bridge.host_binding_ipv4`    | `--ip`      | Default IP when binding container ports               |
 | `com.docker.network.mtu`                         | `--mtu`     | Set the containers network MTU                        |
-| `com.docker.network.enable_ipv6`                 | `--ipv6`    | Enable IPv6 networking                                |
+
+The following arguments can be passed to `docker network create` for any network driver.
+
+| Argument     | Equivalent | Description                              |
+|--------------|------------|------------------------------------------|
+| `--internal` | -          | Restricts external access to the network |
```

Comment:
@thaJeztah yes, this is in the context of the table above which says "and the equivalent docker daemon flags used for docker0 bridge" above it.

@mavenugo true. Want me to remove them? If so, I'll add a sentence above the exiting table saying something like "There are also arguments to `docker network create` applicable to any network driver, which can be found on the reference page for that command."